packages/js-lib/src/zod/zod.shared.schemas.ts

```diff
@@ -88,10 +89,37 @@ export const zIanaTimezone = (): z.ZodEnum =>
     // UTC is added to assist unit-testing, which uses UTC by default (not technically a valid Iana timezone identifier)
     .enum([...Intl.supportedValuesOf('timeZone'), 'UTC'])
 
+export const zBaseDBEntity = (): z.ZodObject<{
```

Comment:
p3: noticed that we export zBaseDBEntity, but not zDBEntity

I don't know what's best, just looked inconsistent